tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -464,8 +532,9 @@ def _LinksAreValid(self, entity: EntityInstance) -> bool:
     entity_type = self.universe.GetEntityType(entity.namespace,
                                               entity.type_name)
     if entity_type and entity_type.allow_undefined_fields and entity.links:
-      print('This entity is not allowed to be the target of links because it is'
-            ' an instance of a passthrough entity type.')
+      print(f'[ERROR]\tEntity {entity.guid} ({entity.code}) is not allowed to '
+            f'be the target of links because it is defined as a passthrough '
+            f'entity.')
```

Comment:
nit: no need for an `f` on the last line?

Reply:
+1